tests/backwards_compatibility_tests/compatibility_test_runner.py

```diff
@@ -366,130 +370,35 @@ def cleanup_volumes():
             logger.debug(f"Warning: Failed to remove volume {volume_name}: {e}")
     volumes_to_cleanup.clear()
 
-def backwards_compatibility_test(from_version: str, to_version: str, to_version_image_idenfitifer: str):
-    """
-    Perform a backwards compatibility test between two versions of Marqo.
-
-    This function starts a container with the from_version, runs tests in prepare mode, stops the container,
-    starts a container with the to_version by transferring state from from_version container, and runs tests in test mode.
-
-    Args:
-        from_version (str): The source version of the Marqo container.
-        to_version (str): The target version of the Marqo container.
-        to_version_image_idenfitifer (str): The unique identifier for a to_version image. It can be either be the fully qualified image name with the tag
-                                (ex: 424082663841.dkr.ecr.us-east-1.amazonaws.com/marqo-compatibility-tests:abcdefgh1234)
-                                or the fully qualified image name with the digest (ex: 424082663841.dkr.ecr.us-east-1.amazonaws.com/marqo-compatibility-tests@sha256:1234567890abcdef).
-                                This is constructed in build_push_image.yml workflow and will be the qualified image name with digest for an automatically triggered workflow.
-
-    Raises:
-        ValueError: If the major versions of from_version and to_version are incompatible.
-        Exception: If there is an error during the test process.
+def run_tests_across_versions(mode: str, from_version: str):
```

Comment:
Both name and description appear misleading because this method doesn't actually perform state transfer and is operating on a single version of marqo only